bin/endtoend-build

```diff
@@ -2,6 +2,14 @@
 
 echo "Preparing for end to end test: copying static HTML and CSS, building JS"
 rm -rf ./test/endtoend/dist && mkdir ./test/endtoend/dist
-cp ./dist/uppy.min.css ./test/endtoend/dist 
-cp ./test/endtoend/src/index.html ./test/endtoend/dist 
+rm -rf ./test/endtoend/node_modules
+
+UPPY_VERSION=$(cat package.json | grep version | head -1 | awk -F= "{ print $2 }" | sed 's/[version:,\",]//g' | tr -d '[[:space:]]')
+# archive the uppy package
+npm pack
```

Comment:
I think we should also run the `prepublishOnly` script here, so that the package is exactly the same as the published one (I thought this would happen automatically inside `npm pack` but it doesn't seem to)

Reply:
done